kura/test/org.eclipse.kura.wire.provider.test/src/main/java/org/eclipse/kura/wire/provider/test/WireServiceTest.java

```diff
@@ -48,36 +48,36 @@
     private static final String receiverPid = "org.eclipse.kura.wire.test.receiver";
 
     /** Configuration Service Reference */
-    private static ConfigurationService s_configService;
+    private static ConfigurationService configService;
 
     /** Logger */
-    private static final Logger s_logger = LoggerFactory.getLogger(WireServiceTest.class);
+    private static final Logger logger = LoggerFactory.getLogger(WireServiceTest.class);
 
     /** Configuration Service Reference */
-    private static WireService s_wireService;
+    private static WireService wireService;
 
     /**
      * Binds the configuration service dependency
      *
-     * @param configService
+     * @param cfgService
      *            the configuration service dependency
      */
-    public void bindConfigService(final ConfigurationService configService) {
-        if (s_configService == null) {
-            s_configService = configService;
+    public void bindConfigService(final ConfigurationService cfgService) {
+        if (configService == null) {
+            configService = cfgService;
             dependencyLatch.countDown();
         }
     }
 
     /**
      * Binds the wire service dependency
      *
-     * @param wireService
+     * @param wreService
      *            the wire service dependency
      */
-    public void bindWireService(final WireService wireService) {
-        if (s_wireService == null) {
-            s_wireService = wireService;
+    public void bindWireService(final WireService wreService) {
+        if (wireService == null) {
```

Comment:
I believe you can apply the formatter/cleanup profiles that will set the `this.something `as needed. In this way, we could keep the wireService variable as is and not change it as `wreService` that seems more a misspelt variable.